fedimint-core/src/encoding/threshold_crypto.rs

```diff
@@ -0,0 +1,121 @@
+use std::io::{Error, Read, Write};
+
+use threshold_crypto::group::Curve;
+use threshold_crypto::{G1Affine, G1Projective};
+
+use crate::encoding::{Decodable, DecodeError, Encodable};
+use crate::module::registry::ModuleDecoderRegistry;
+
+impl Encodable for threshold_crypto::PublicKeySet {
```

Comment:
Can't we just derive all this now? Care would need to be taken not to break anything, but worth looking into imo (follow-up, not this PR).